ddtrace/internal/constants.py

```diff
@@ -69,6 +69,7 @@
 EXTERNAL_ENV_ENVIRONMENT_VARIABLE = "DD_EXTERNAL_ENV"
 
 MESSAGING_SYSTEM = "messaging.system"
+MESSAGING_DESTINATION_NAME = "messaging.destination.name"
```

Comment:
Will this tag be unique to the kafka integration? If so we can move it to `ddtrace/ext/kafka.py`

Reply:
I think it's good to keep it in the general constants file since Otel expects this for general messaging queue attributes: https://opentelemetry.io/docs/specs/semconv/attributes-registry/messaging/ so what @robcarlan-datadog has looks good to me!

dd-trace-dotnet uses the same tag for Azure Service Bus https://github.com/DataDog/dd-trace-dotnet/blob/c025d25e077bf02e8d9025c7dc49fb25187ecde9/tracer/src/Datadog.Trace/Activity/Handlers/AzureServiceBusActivityHandler.cs#L65

Example snapshot: https://github.com/DataDog/dd-trace-dotnet/blob/c025d25e077bf02e8d9025c7dc49fb25187ecde9/tracer/test/snapshots/AzureServiceBusTests_7_17.SchemaV0.verified.txt#L88